src/resmom/mom_comm.c

```diff
@@ -4329,6 +4329,36 @@ im_request(int stream, int version)
 			log_event(PBSEVENT_JOB, PBS_EVENTCLASS_JOB, LOG_DEBUG,
 				jobid, "RESTART received");
 
+			/* NULL value passed to hook_input.vnl */
+			/* means to assign */
+			/* vnode list using pjob->ji_host[].   */
+			mom_hook_input_init(&hook_input);
```

Comment:
Our coding standard suggests use of a multi-line comment instead of multiple single line comments for one comment.  Could you use a multi-line comment here and other places?